### PR TITLE
fix(core): update on changed DMSS api

### LINKED ISSUE #23: chore: update dm-core with breaking change dmss v0.3.23

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Update some functions to reflect the  DMSS endpoints that has changed
- Reverts some changes merged into main after https://github.com/equinor/dm-core-packages/commit/5f3b9bc4048c56c03f0b8c208be70e4e13a9fdcb as the core package was never published, and so all bumping refering to this unpublished packaged failed (@KristianKjerstad 

## Why is this pull request needed?
- DMSS api changed

## Issues related to this change
closes #23

### CODE REVIEW

File: packages/blueprint/package.json [hunk 1]
```diff
@@ -4,18 +4,18 @@
   "version": "1.0.2",
```
Comment (netr0m) [inline packages/blueprint/package.json:4]:
```suggestion
  "version": "1.0.3",
```
Should we bump versions on changes to dependencies?

**Review by KristianKjerstad — APPROVED** [review]
good!

opening up issue https://github.com/equinor/dm-core-packages/issues/21 again

**Review by netr0m — APPROVED** [review]
LGTM!

File: packages/blueprint/package.json (hunk 1 above)
Comment (soofstad) [reply packages/blueprint/package.json:4]: depends, if the changed dependency effect the package so much is validates bumping its version I'd say yes :)

Comment (soofstad) [reply packages/blueprint/package.json:4]: But if you look closely, no dependencies has actually changed, only reordered.
yarn...smh

**Pull request merged**